src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => (
+  <ul>
+    {tabs.map((tab) => {
+      const { id, title } = tab;
```

Comment:
```suggestion
    {tabs.map(({ id, title }) => {
```
you could destructure argument of callback